alphadia/transferlearning/train.py

```diff
@@ -599,14 +610,17 @@ def finetune_ms2(
         self.early_stopping.reset()
 
         # Test the model before training
-        self._test_ms2(
-            -1,
-            0,
-            reordered_val_psm_df,
-            reordered_val_intensity_df,
-            test_metric_manager,
-            data_split="validation",
-        )
+        if set(self.charged_frag_types).issubset(
```

Comment:
Do we need this check here? If yes, should there be an error message if this is not true?